tasks.py

```diff
@@ -455,7 +455,7 @@ def regenerate(
         "low_level_client" if low_level_client else "",
         "legacy" if legacy else "",
     ]
-    generators = [g for g in generators if g] or ["legacy", "low_level_client", "version_tolerant"]
+    generators = [g for g in generators if g] or ["legacy", "version_tolerant"]
```

Comment:
`inv regenerate` will not run `low-level` as default behavior. Since we removed low-level from test framework, the low-level test may become addled gradually. So is it better to remove `test/.../low-level` and relevant code in `tasks.py`? @iscai-msft

Reply:
I think we can get rid of all `low-level` folders and all low level generation in this file. This is a great start though!

Reply:
I like `delete`(ha ha).